locales-src/asrouter.ftl

```diff
@@ -90,6 +90,13 @@ cfr-whatsnew-button =
   .label = What’s New
   .tooltiptext = What’s New
 
+cfr-whatsnew-tracking-protections-title =
```

Comment:
Localizers need the comment, otherwise they won't know what's happening here, and to not add the variable again. You can copy and adapt the one I linked to.

Reply:
@flodolo I'm not sure if the existing pattern works for this case of a title that goes before the number. Or at least it might want some extra clarification to localizers. The protections page example has numbers that appear immediately before the text:
![Screen Shot 2019-08-22 at 12 37 04 AM](https://user-images.githubusercontent.com/438537/63495763-55ba4800-c475-11e9-8251-9572a69488ca.png)

Here's the romanian strings for "Passwords exposed across all breaches.":
https://pontoon.mozilla.org/ro/firefox/browser/browser/protections.ftl/?string=201515
1 parolă …
2 parole …
20 de parole …

@piatra says a title starting with "De …" doesn't make sense, e.g.,
```
De parole …
20
```
while this is okay:
```
Parole …
20
```

I suppose romanian localization could just repeat an entry without "De ":
```
{ $blockedCount ->
  [one] Parolă …
  [few] Parole …
  [other] Parole …
```

The localization note already says "displayed **before** a large numeral…" so I maybe that might be enough information to decide to drop the "de " ??

Here's an example without the leading "de" that would normally go between the "other" number and the noun:
![Screen Shot 2019-08-22 at 1 00 58 AM](https://user-images.githubusercontent.com/438537/63497377-7506a480-c478-11e9-8fb2-de28b224932a.png)

I think even for english "Trackers Blocked: 1" makes sense. E.g., in a full sentence: "Here's how many trackers Firefox blocked: 1"

Reply:
> I'm not sure if the existing pattern works for this case of a title that goes before the number.

The comment clearly says that the numeral is before the string. Each locale will have the chance to decide if they need to decline the noun or use a generic plural and singular.